src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyFollowedBySemicolonHasLineBreakAfter.java

```diff
@@ -0,0 +1,15 @@
+/*
+RightCurly
+option = ALONE_OR_SINGLELINE
+tokens = LITERAL_TRY , LITERAL_CATCH , LITERAL_FINALLY , LITERAL_IF , LITERAL_ELSE , CLASS_DEF , METHOD_DEF , CTOR_DEF , LITERAL_FOR , LITERAL_WHILE , LITERAL_DO , STATIC_INIT , INSTANCE_INIT , ANNOTATION_DEF , ENUM_DEF , INTERFACE_DEF , RECORD_DEF , COMPACT_CTOR_DEF
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.blocks.rightcurly;
+
+public class InputRightCurlyFollowedBySemicolonHasLineBreakAfter {
+
+        Object o = new Object() {
+        }; int i = 0;  // violation '';' at column 10 should have line break after'
```

Comment:
Here my idea was that I should check for a line break after the semicolon which is right after the rightcurly brace.